android/app/src/main/java/org/electroncash/electroncash3/ColdLoad.kt

```diff
@@ -179,26 +170,27 @@ fun canBroadcast(tx: PyObject): Boolean {
 /**
  * Sign a loaded transaction dialog.
  */
-class SignPasswordDialog : PasswordDialog<Unit>() {
+class SignPasswordDialog : PasswordDialog<PyObject>() {
     val coldLoadDialog by lazy { targetFragment as ColdLoadDialog }
-    val signSchnorr = daemonModel.walletType == "standard"
+    // Schnorr signing is supported by standard and imported private key wallets.
+    val signSchnorr = daemonModel.walletType in listOf("standard", "imported_privkey")
```

Comment:
This duplicates the code in Send.kt. Even if it's only one line, it should be broken out into a function, so that if needs to change in the future there will only be one place to edit.